reporting_database_generator/create_reporting_db.py

```diff
@@ -0,0 +1,25 @@
+from lib.configuration import get_connection_string, get_required_tables, get_current_config
+from sqlalchemy import create_engine
+import datetime
+
+def reporting_db_creation(dbtype='granted_patent' , **kwargs):
+    from lib.configuration import get_current_config
+    config = get_current_config(dbtype, schedule="quarterly", **kwargs)
+    ed = config['DATES']['END_DATE']
```

Comment:
Why not use the execution/logical date here as well?

Reply:
This function isn't referenced anymore because I did just that with the SQL